### DISCUSSION TITLE
Translation formals

### DISCUSSION THREAD

**Discussion by ghost** [astarub/campus_app#53] [Q&A] [answered]
Should translations also change the @@locale Attribute (this is not mentioned in the doku)
https://github.com/astarub/campus_app/wiki/Translation

Also should there be seperate translations for german and german formal or is there an idea if it should be formal or informal german only?

**Comment by BizarrePenguin** [comment]
Imo only informal language should be used. I think it feels a lot more natural, especially for students.

**Reply by BizarrePenguin** [reply]
I added an informal german translation file in PR #55.

**Reply by ghost** [reply]
Im unhappy with some of your translations and fixed a small thing in the english one in https://github.com/astarub/campus_app/pull/56

**Comment by henry-herrmann** [comment, marked as answer]
Yeah translations should definitely change the locale attribute.

One translation whether it's formal, informal or a mixture of both is certainly best suited in most cases.

**Reply by ghost** [reply]
thanks for the clarification. I assume we stick to informal german then? Also is there a milestone for adding the translation for more elements like "Kosten" in currency Tab like Descriptions in the Guide or other Elements like Allergene? Also i think it would be a good idea to add more context to the translation files maybe document how they are used for translators. for example login can mean the activity but also the noun. It would be nice to have more descriptive elements or to specify the translations further in a wiki entry.

**Reply by henry-herrmann** [reply]
Yeah translations definitely have to be worked on since almost everything is hard coded. I'll have it discussed in the next meeting. Thank you.

**Reply by Nicole4Fox** [reply]
Anything new from the discussion of that meeting?
